### PR TITLE
Ref #58: Create 404.html

### PR DESCRIPTION
**Pull request by cjanzen**
Create a custom 404 file to replace the current one. I'm not sure the links in the ul are working correctly, but I wasn't able to test from my local. Refers to #58

### CODE REVIEW

File: _includes/404.html [hunk 1]
```diff
@@ -14,10 +14,10 @@ <h2 class="section-heading">Error 404 - Page not Found!</h2>
...
-                <li>Who's talking it up: <a href="#speakers.html">Speakers</a> or <a href="#previous.html">Alumni</a></li>
-                <li>Who's supporting us: <a href="#sponsor.html">Sponsor</a></li>
-                <li>How to help: <a href="#volunteer.html">Volunteer</a></li>
+                <li>What's going on: <a href="http://www.wichitalks.com/#events.html">Events</a></li>
+                <li>Who's talking it up: <a href="http://www.wichitalks.com/#speakers.html">Speakers</a> or <a href="http://www.wichitalks.com/#previous.html">Alumni</a></li>
+                <li>Who's supporting us: <a href="http://www.wichitalks.com/#sponsor.html">Sponsor</a></li>
+                <li>How to help: <a href="http://www.wichitalks.com/#volunteer.html">Volunteer</a></li>
             </ul>
           </div>
         <p>You can also use your browser's back button to return to your previous page or go <a href="http://www.wichitalks.com" target="_blank">home</a> to start again.</p>
```
Comment (kevinfalting) [inline _includes/404.html]: We don't need to open a new tab for them to go back to home. You can remove the target="_blank".

File: _includes/404.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<section id="404">
+    <div class="container">
+        <div class="row">
+            <div class="col-lg-12 text-center">
+                <h2 class="section-heading">Error 404 - Page not Found!</h2>
+                <hr class="primary">
+                <p>Oh, snap! Clearly, this is not the page you're looking for. The page may have been renamed or removed.</p>
```
Comment (kevinfalting) [inline _includes/404.html:7]: Lol, thanks for the cool message.

Comment (cjanzen) [reply _includes/404.html:7]: Thanks. :) I tried to make the tone of the writing similar to the rest of the site.

File: _includes/404.html (hunk 1 above)
Comment (cjanzen) [reply _includes/404.html]: Rats, I thought I got them all. I'll fix the one that sends the user home again. Sorry about that.

**Comment by cjanzen** [comment]
Made the change requested. Fingers crossed that all is ready. :)

**Pull request merged**